addons/stock/models/stock_move.py

```diff
@@ -893,6 +902,14 @@ def _onchange_product_id(self):
         product = self.product_id.with_context(lang=self._get_lang())
         self.name = product.partner_ref
         self.product_uom = product.uom_id.id
+        if self.user_has_groups('product.group_stock_packaging'):
```

Comment:
not needed to check packaging group everywhere. It shouldn't arrive filled if the group is not active